src/Stripe/Services/Subscriptions/StripeSubscriptionCreateOptions.cs

```diff
@@ -42,6 +42,24 @@ internal string TrialEndInternal
         [JsonProperty("tax_percent")]
         public decimal? TaxPercent { get; set; }
 
+        public DateTime? BillingCycleAnchor { get; set; }
+
+        public bool BillingCycleAnchorNow { get; set; }
+
+        [JsonProperty("billing_cycle_anchor")]
+        internal string BillingCycleAnchorInternal
+        {
+            get
+            {
+                if (EndTrialNow)
```

Comment:
yes, a copy paste bug, can you fix it when you merge it? I can send a new pull request otherwise. Thanks!